### PR TITLE
TJSW pre-oop task

### PR DESCRIPTION
**Pull request by JuliaNovak**
### TJSW pre-oop task

[Demo](https://julianovak.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/JuliaNovak/TJSW-pre-oop-task/blob/main/index.js)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by JuliaNovak** [comment]
have completed self-checks and fixed code accordingly

File: submissions/JuliaNovak/TJSW-pre-oop/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+   saying: "Heyyo!",
+};
+
+const catWoman = {
+   species: "cat-woman",
+   name: "Ro",
+   gender: "female",
+   legs: 2,
+   hands: 2,
+   saying: cat.saying,
```
Comment (artem-trubin) [inline submissions/JuliaNovak/TJSW-pre-oop/index.js:45]: Task requires your catWoman's saying to always say what cat says. If your cat saying changes, your catwoman's doesn't. It only stores the initial `cat.saying` value.

**Review by artem-trubin — CHANGES_REQUESTED** [review]
Hi, your `Code base` link leads to the `Files changed` tab of this PR. It should lead to your original code base, it means your personal repository.

Please, check the comments below.

File: submissions/JuliaNovak/TJSW-pre-oop/index.js (hunk 1 above)
Comment (JuliaNovak) [reply submissions/JuliaNovak/TJSW-pre-oop/index.js]: fixed

File: submissions/JuliaNovak/TJSW-pre-oop/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+   name: "Jim",
+   gender: "male",
+   legs: 2,
+   hands: 2,
+   saying: "Heyyo!",
+};
+
+const catWoman = Object.create(cat);
+catWoman.species = "catWoman";
+(catWoman.name = "Ro"), (catWoman.legs = 2), (catWoman.hands = 2);
```
Comment (artem-trubin) [inline submissions/JuliaNovak/TJSW-pre-oop/index.js:41]: Interesting approach, but it is fine to write this in 3 lines. It's not like it's some code that could be "skipped", it's an important value assigning that describes one of the objects.

**Review by artem-trubin — APPROVED** [review]

File: submissions/JuliaNovak/TJSW-pre-oop/index.js (hunk 2 above)
Comment (JuliaNovak) [reply submissions/JuliaNovak/TJSW-pre-oop/index.js:41]: is there another way to do it?

Comment (artem-trubin) [reply submissions/JuliaNovak/TJSW-pre-oop/index.js:41]: Just write 3 separate lines)

Comment (JuliaNovak) [reply submissions/JuliaNovak/TJSW-pre-oop/index.js:41]: I mean, is there a way to do it without using Object.create?:)

**Pull request merged**